scriptworker/constants.py

```diff
@@ -176,14 +176,16 @@
     # Map scopes to restricted-level
     'cot_restricted_scopes': frozendict({
         'firefox': frozendict({
-            'project:releng:balrog:release': 'all-release-branches',
-            'project:releng:beetmover:release': 'all-release-branches',
+            'project:releng:balrog:server:release': 'all-release-branches',
```

Comment:
Ok.  For reference, I added the new balrog/beetmover scopes in [this patch](https://reviewboard.mozilla.org/r/113722/diff/2#index_header).